components/menu/calendar/Screens/FetchScreen/index.js

```diff
@@ -1,22 +1,71 @@
 import React, { Component } from 'react';
 import { View, AsyncStorage, ActivityIndicator } from 'react-native';
+import CheckboxGroup from './CheckboxGroup';
 
 export default class FetchScreen extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      calendarsGeneralInfo: [],
+    };
+    //this.getDatta=this.getDatta.bind(this);
+  }
+
   componentDidMount() {
-    this.getData();
+    //console.log('length:');
+    console.log(this.state.calendarsGeneralInfo.length);
+    this.getDatta();
+    if (this.state.calendarsGeneralInfo.length === 0) {
+      //console.log('am here');
+      this.forceUpdate();
+    }
+    console.log('--!!!_--~~~~-  The moment of truth ');
+    this.asyncStorageChecker();
   }
 
-  getData = async () => {
+  // eslint-disable-next-line consistent-return
```

Comment:
please remove this, and add return null statement under the if case

Reply:
Has been removed.